atomics/T1560.001/T1560.001.yaml

```diff
@@ -276,3 +276,41 @@ atomic_tests:
       ls -l #{test_folder}
     cleanup_command: |
       rm -Rf #{test_folder}
+- name: 'Encrypts collected data with AES-256 and Base64'
+  auto_generated_guid: 
```

Comment:
GUIDs will be auto generated. You can remove this one too.

Reply:
Done! :)

Reply:
Thank you.